### PR TITLE
Add step in ops to install services declarations dependencies

### PR DESCRIPTION
**Pull request by Ndpnt**
Fix bug on prod where services filters dependencies are missing.

### CODE REVIEW

**Comment by martinratinaud** [comment]
Not sure but think it's a dup of https://github.com/ambanum/OpenTermsArchive/issues/716#issuecomment-986740230 where I tried something which did not work (for reference)

File: ops/roles/ota/tasks/main.yml [hunk 1]
```diff
@@ -37,6 +37,14 @@
     - setup
     - update
 
+- name: Install services declarations dependencies
+  command:
+    cmd: npm i --production --production=true
```
[the remark this answers is not in the document]
Comment (martinratinaud) [reply ops/roles/ota/tasks/main.yml:43]: @Ndpnt stil do not know why you need it twice

**Review by martinratinaud — APPROVED** [review]
I would include in this PR the fact of deleting and recreating pm2 

https://github.com/ambanum/OpenTermsArchive/commit/90c2c1cd8c23bab2858d786b56711e39d37ecbab

File: ops/roles/ota/tasks/main.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Ndpnt) [reply ops/roles/ota/tasks/main.yml:43]: Why twice?

[the remark this answers is not in the document]
Comment (Ndpnt) [reply ops/roles/ota/tasks/main.yml:43]: The first one install OTA dependencies and the second one install services declarations dependencies

[the remark this answers is not in the document]
Comment (martinratinaud) [reply ops/roles/ota/tasks/main.yml:43]: please add a comment because I still do not get it 🤣

[the remark this answers is not in the document]
Comment (Ndpnt) [reply ops/roles/ota/tasks/main.yml]: Sorry I read too fast. Fixed now

**Pull request merged**